### DISCUSSION TITLE
development env error

### DISCUSSION THREAD

**Discussion by gaecoli** [getredash/redash#6216] [General]
<img width="914" alt="image" src="https://github.com/getredash/redash/assets/46274612/9400cde3-3d0f-4474-b84d-6c0c096898e0">

I install command:
`pip install -r requirements.txt -r requirements_dev.txt`
`rq worker`

**Comment by justinclift** [comment]
Oh, that's the SIGRTMIN problem.  Are you running on macOS?

**Reply by justinclift** [reply]
If you are, then this will probably fix it: https://github.com/getredash/redash/pull/5513

**Reply by gaecoli** [reply]
Yes. I know how to fix it. I will fix it.

**Reply by gaecoli** [reply]
Fixed it!
